src/service-broker.js

```diff
@@ -834,8 +839,8 @@ class ServiceBroker {
 			service = new this.ServiceFactory(this, schema, schemaMods);
 		}
 
-		// If broker has started yet, call the started lifecycle event of service
-		if (this.started) this._restartService(service);
+		// If broker has began to start its initial services yet, call the started lifecycle event of service
+		if (this.servicesStarted) this._restartService(service);
```

Comment:
the `if (this.started || this.servicesStarting)` would be better.

Reply:
updated